src/v2/guide/migration.md

```diff
@@ -1257,57 +1257,65 @@ Los enlaces _one-time_ (`{% raw %}{{* foo }}{% endraw %}`) han sido reemplazados
 </div>
 {% endraw %}
 
-## Reactivity
+## Reactividad
 
-### `vm.$watch` <sup>changed</sup>
+### `vm.$watch` <sup>cambiado</sup>
 
-Watchers created via `vm.$watch` are now fired before the associated component rerenders. This gives you the chance to further update state before the component rerender, thus avoiding unnecessary updates. For example, you can watch a component prop and update the component's own data when the prop changes.
+Los _watchers_ creados a través de `vm.$watch` ahora se disparan antes de que el componente asociado se vuelva a renderizar. Esto le brinda la oportunidad de actualizar el estado antes de que el componente vuelva a renderizarse, evitando así actualizaciones innecesarias. Por ejemplo, puede escuchar por una propiedad y actualizar sus propios datos cuando la propiedad cambia.
```

Comment:
I think that `escuchar` is more related to events than watchers, what do you think about changing to `observar`:

```suggestion
Los _watchers_ creados a través de `vm.$watch` ahora se disparan antes de que el componente asociado se vuelva a renderizar. Esto le brinda la oportunidad de actualizar el estado antes de que el componente vuelva a renderizarse, evitando así actualizaciones innecesarias. Por ejemplo, puede observar una propiedad y actualizar sus propios datos cuando la propiedad cambia.
```